inc/autoload/class-add-plugin-list.php

```diff
@@ -209,9 +210,13 @@ public function manage_plugins_custom_column( $column_name, $plugin_file, $plugi
 						$class = ' class="site-archived"';
 						$hint  = esc_attr__( ', Archived site', 'multisite-enhancements' );
 					}
+					elseif ( $this->is_deleted( $key ) ) {
+						$class = ' class="site-deleted"';
+						$hint  = esc_attr__( ', Deleted site', 'multisite-enhancements' );
+					}
 					$output .= '<li' . $class . ' title="Blog ID: ' . $key . $hint . '">';
 					$output .= '<nobr><a href="' . get_admin_url( $key ) . 'plugins.php">'
-					. $value['name'] . '</a>' . $hint . '</nobr></li>';
+					. ( trim( $value['name'] ) ?: $value['path'] ) . '</a>' . $hint . '</nobr></li>';
```

Comment:
The nobr tag is deprecated in html, maybe we should use this PR as chance to replace them with span and css.

Reply:
Sure! Done in https://github.com/bueltge/wordpress-multisite-enhancements/pull/62/commits/5a927603d73209bcd3b21b26871ccef7ecf15c90